lib/generators/shopify_app/install/install_generator.rb

```diff
@@ -7,17 +7,27 @@ class InstallGenerator < Rails::Generators::Base
       include Rails::Generators::Migration
       source_root File.expand_path('../templates', __FILE__)
 
+      class_option :application_name, type: :string, default: nil
```

Comment:
Can you verify what the default is if you don't include `default: nil`? If it's already `nil` then you don't need to make it explicit.